fiware_cloto/environments/environment_controller.py

```diff
@@ -35,23 +35,38 @@ class environment_controller():
 
     started = False
 
-    def check_python_process(self):
-        p = subprocess.Popen(['ps', '-awx'], stdout=subprocess.PIPE)
-        out, err = p.communicate()
 
-        for line in out.splitlines():
-            if 'environmentManager.py' in line:
-                self.started = True
-
-        return self.started
 
     def start_manager(self):
         if settings.SETTINGS_TYPE == 'production':
             arbiter = get_arbiter([{"cmd": "python "
                                        "" + settings.ENVIRONMENTS_MANAGER_PATH, "numprocesses": 1}], background=True)
 
-            if not self.check_python_process():
+            if not check_python_process():
+                arbiter.start()
+            else:
+                clean_environments()
                 arbiter.start()
 
     def is_started(self):
         return self.started
+
+
+def clean_environments():
+        cmd = "ps -awx | grep fiware_cloto/environments | awk '{print $1}' |xargs kill -9"
```

Comment:
I have a question regarding this operation, if I execute this sentence automatically I create a process like:

44657 ttys000    0:00.00 grep fiware_cloto/environments

so, you always will find a pid in the execution of this line whenever you have the environmentManager.py process or not.